--- /usr/local/lib/python3.6/site-packages/ansible/modules/net_tools/nmcli.py	2019-09-25 18:12:24.396858739 +0200
+++ /usr/local/lib/python3.6/site-packages/ansible/modules/net_tools/nmcli.py	2019-09-25 18:14:48.695797176 +0200
@@ -22,12 +22,12 @@
 short_description: Manage Networking
 requirements:
 - dbus
-- NetworkManager-glib
+- NetworkManager-libnm
 - nmcli
 version_added: "2.0"
 description:
     - Manage the network devices. Create, modify and manage various connection and device type e.g., ethernet, teams, bonds, vlans etc.
-    - 'On CentOS and Fedora like systems, the requirements can be met by installing the following packages: NetworkManager-glib,
+    - 'On CentOS and Fedora like systems, the requirements can be met by installing the following packages: NetworkManager-libnm,
       libnm-qt-devel.x86_64, nm-connection-editor.x86_64, libsemanage-python, policycoreutils-python.'
     - 'On Ubuntu and Debian like systems, the requirements can be met by installing the following packages: network-manager,
       python-dbus (or python3-dbus, depending on the Python version in use), libnm-glib-dev.'
@@ -369,7 +369,7 @@
   - name: install needed network manager libs
     package:
       name:
-        - NetworkManager-glib
+        - NetworkManager-libnm
         - nm-connection-editor
         - libsemanage-python
         - policycoreutils-python
@@ -565,10 +565,9 @@
 NM_CLIENT_IMP_ERR = None
 try:
     import gi
-    gi.require_version('NMClient', '1.0')
-    gi.require_version('NetworkManager', '1.0')
+    gi.require_version('NM', '1.0')
 
-    from gi.repository import NetworkManager, NMClient
+    from gi.repository import NM
     HAVE_NM_CLIENT = True
 except (ImportError, ValueError):
     NM_CLIENT_IMP_ERR = traceback.format_exc()
